lib/options.js

```diff
@@ -104,15 +91,32 @@ exports.parse = function parse(argv) {
     else if(option === '--no-profile') {
       options.profile = false;
     }
-    else if(option === '--no-channel') {
-      options.channel = false;
-    }
-    else if({'--cluster':1,'--port':1,'--addr':1,'--path':1}[option]) {
-      i++; // Consume the option's argument
+    else if(option === '--cluster') {
+      i++;
+      cluster = argv[i];
     }
-    else if(/^--(cluster|port|addr|path)=/.test(option)) {
+    else if(/^--cluster=/.test(option)) {
+      cluster = value(argv[i], cluster);
     }
     else if(option === '--no-cluster') {
+      cluster = 'off';
+    }
+    // Only -C,--control,--no-control are documented, now, to align with
+    // strong-pm. The others are legacy, to be deleted sometime.
+    else if({
+      '-C':1, '--control':1, '--port':1, '--addr':1, '--path':1
```

Comment:
I think I commented on this elsewhere, but these look like default values rather than "arbitrary truthy values", as opposed to the settings below that are explicitly `false` instead of `0`.